test/EFCore.PG.FunctionalTests/EFCore.PG.FunctionalTests.csproj

```diff
@@ -19,6 +19,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\EFCore.PG\EFCore.PG.csproj" />
+    <ProjectReference Include="..\..\src\EFCore.PG.NTS\EFCore.PG.NTS.csproj" />
```

Comment:
Hmm.... Should we consider going to a single test suite project? Or is this overlap just for integration, rather than functionality testing?

Reply:
So we have an EFCore.PG.Plugins.FunctionalTests, which before this PR contained all tests for the plugins. The main idea in having a separate project was for building speed - to avoid having a dependency from EFCore.PG.FunctionalTests on the plugin projects, and so we don't have to compile them every time. However, with the new EF Core spatial support spatial tests are included in the main FunctionalTests suite, so a dependency has to be taken on the NTS plugin anyway.

I actually looked at consolidating to have just one FunctionalTests project, but ran into an unfortunate problem: the NodaTime plugin modifies the ADO type mappings globally, and this breaks all the other date/time tests. Having the NodaTime plugin tests in their own projects isolates the main tests from this, which is a pretty crappy workaround but works for now...

Reply:
Ahhh. Okay, that makes sense. It does also seem reasonable that over time there may be plugins more closely associated with the upstream project than others.

Perhaps something to revisit if we find a more nuanced type registration/resolution pattern. (We've had a few issues opened related to this, so it might be worth some time in the next release.)